Atomic/atomic.py

```diff
@@ -272,6 +360,11 @@ def set_args(self, args):
             self.spc = False
 
         try:
+            self.system = args.system
+        except:
+            self.system = False
```

Comment:
No big thing, but I thought we were defaulting self.system=false in the init, so that if this throws the exception, it will still be set to false.  Which is why I suggested to just pass it.

```
          try:
            self.system = args.system
        except:
            pass
```